component/booksCard.jsx

```diff
@@ -0,0 +1,57 @@
+import useStore from '../hooks/useStore';
+import { useEffect } from 'react';
+import styled from 'styled-components';
+
+const Api = process.env.NEXT_PUBLIC_REACT_APP_API_KEY;
```

Comment:
Highly recommend using uppercase characters for this kind of constant.
```suggestion
const API = process.env.NEXT_PUBLIC_REACT_APP_API_KEY;
```
And by the way, are you using this anywhere here?